src/api/Feed.js

```diff
@@ -979,7 +980,7 @@ class Feed extends Base {
      */
     feedErrorCallback = (hasError: boolean = false, e: ElementsXhrError, code: string): void => {
         if (hasError) {
```

Comment:
It looks like this is only true if the error is considered "user-correctable". Does that cover our use cases?

Reply:
I actually don't understand the concept "user-correctable". Our use cases is fetch error, the possible reason might be timeout (504) or service unavailable (503). I looked into the code and if the error code is larger than 500, `hasError` will be true. So I think it covers our use cases.